pkg/kubelet/dockershim/docker_checkpoint.go

```diff
@@ -82,19 +85,24 @@ func NewPodSandboxCheckpoint(namespace, name string, data *CheckpointData) Docke
 	}
 }
 
+// MarshalCheckpoint encode the PodSandboxCheckpoint instance to a json object
```

Comment:
Small nit: For `MarshalCheckpoint`, `UnmarshalCheckpoint`, and `GetData`, I think we want `MarshalCheckpoint encodes`, `UnmarshalCheckpoint decodes`, and `GetData gets`.